### PR TITLE
Add YveltalNL

### PR DESCRIPTION
**Pull request by WarriorGallade**

### CODE REVIEW

**Comment by HisuianZoroark** [comment]
we dont typically have things bleed out into files in sim/ 

for now use ``this.species.get(target.species).heightm`` for height getting

**Review by HisuianZoroark — CHANGES_REQUESTED** [review]
see above.

File: data/mods/gen9ssb/moves.ts [hunk 1]
```diff
@@ -2554,6 +2554,36 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		volatileStatus: "highground",
+		condition: {
+			onStart(target, source, sourceEffect) {
+				if (source.getHeight() > target.getHeight()) {
+					this.boost({spa: 1}, source);
+				}
+			},
+		},
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/moves.ts:2574]: a volatile is the wrong thing to use here

Comment (WarriorGallade) [reply data/mods/gen9ssb/moves.ts:2574]: changing to onHit, reverting the /sim changes in a bit too. tbf it is odd how sim has no way to get height already, even if no mechanic uses it

File: data/mods/gen9ssb/moves.ts [hunk 2]
```diff
@@ -2554,6 +2554,34 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		secondary: {
+			onHit(target, source, move) {
+				if (this.dex.species.get(source.species).heightm > this.dex.species.get(target.species).heightm) {
+					this.boost({spa: 1}, source);
+				}
+			},
+		},
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/moves.ts:2573]:
```suggestion
		secondary: {
			chance: 100,	
			onHit(target, source, move) {
				if (this.dex.species.get(source.species).heightm > this.dex.species.get(target.species).heightm) {
					this.boost({spa: 1}, source);
				}
			},
		},
```

**Pull request merged**